setup.py

```diff
@@ -6,7 +6,7 @@
 
 install_requires = [
     'cffi',
-    'chainer >=3.3.0, !=4.0.0b2, != 4.0.0b1, != 4.0.0a1',
+    'chainer >=3.5.0, !=4.0.0b2, != 4.0.0b1, != 4.0.0a1',
```

Comment:
Do we need to exclude 4.0.0rc1 as well?

Reply:
I don't think so because it works. See also: https://github.com/chainer/chainermn/pull/233/files#diff-1ed392351c4f47516bae8214681548eeR27

Reply:
Got it!